src/glyph/mod.rs

```diff
@@ -251,6 +252,49 @@ impl Contour {
     fn is_closed(&self) -> bool {
         self.points.first().map_or(true, |v| v.typ != PointType::Move)
     }
+
+    /// Converts the contour to a kurbo::BezPath
+    #[cfg(feature = "kurbo")]
+    pub fn to_kurbo(&self) -> Result<kurbo::BezPath, ErrorKind> {
+        let mut path = kurbo::BezPath::new();
+        let mut offs: VecDeque<kurbo::Point> = VecDeque::new();
+        // Add end-of-contour offcurves to queue
+        let mut i = self.points.len() - 1;
+        while i > 0 && self.points[i].typ == PointType::OffCurve {
+            let kurbo_point = kurbo::Point::new(self.points[i].x as f64, self.points[i].y as f64);
+            offs.push_front(kurbo_point);
+            i -= 1;
+        }
```

Comment:
A few thoughts:

- are we okay with encoding the assumption that the start point in a closed path is always the last on-curve in the point list? I think this is a good convention, but it is not enforced in the spec.
- If we're doing that, I would probably try to write this all in terms of an iterator? I try to avoid indexing into collections where I can avoid it, because it is harder to reason about (any invalid index is a runtime panic) and also can have performance costs (if the compiler doesn't know an index is valid, it has to do a bounds check each time).

With that in mind, I might try to do something like,

```rust
let mut points = if self.is_closed() {
    let rotate = self.points.iter().rev().position(|pt| pt.typ != PointType::OffCurve).map(|idx| self.points.len() - 1 - idx);
    // is this an off-by-one? we want to see the first point twice
    self.points.iter().cycle().skip(rotate.unwrap_or(0)).take(self.points.len() + 1)
} else {
   // we need to return the same concrete iterator type from both branches
   self.points.iter().cycle().skip(0).take(self.points.len())
}
if let Some(start) = points.next() {
    path.move_to(start);
}
for pt in points {
// the usual logic
}
```

In any case this is mostly code golf, and definitely a matter of preference, but it has the nice quality that once we get the points in the right order we just build the path out of a single run through the iterator.